src/tests/BuildWasmApps/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -230,6 +234,12 @@ public BuildTestBase(ITestOutputHelper output, SharedBuildPerTestClassFixture bu
                                         timeoutMs: s_defaultPerTestTimeoutMs);
 
             File.WriteAllText(Path.Combine(testLogPath, $"xharness.log"), output);
+            if (useWasmConsoleOutput)
+            {
+                string wasmConsolePath = Path.Combine(testLogPath, "wasm-console.log");
+                if (File.Exists(wasmConsolePath))
+                    output = File.ReadAllText(wasmConsolePath);
```

Comment:
if it doesn't exist, maybe emit a warning to specify that we looked for the log and it wasn't there?

Reply:
this read could also hit a sharing violation if the test process has it locked too, i think? i'm not sure. might be worth having a try-catch around it